README.md

```diff
@@ -110,26 +113,80 @@ Unity version 4.6.8 or higher.
 The *Version Handler* component only works with Unity 5.x or higher as it
 depends upon the `PluginImporter` UnityEditor API.
 
-# Android Resolver
+# Getting Started
+
+Before you import the Play Services Resolver into your plugin project, you first 
+need to consider whether you intend to *redistribute* Play Services Resolver
+along with your own plugin.
+
+Redistributing the `Play Services Resolver` inside your own plugin will ease 
+the integration process for your users, by resolving dependency conflicts
+between your plugin and other plugins in a user's project.
+
+If you wish to redistribute the `Play Services Resolver` inside your plugin,
+you **must** follow these steps when importing the 
+`play-services-resolver-*.unitypackage`, and when exporting your own plugin
+package:
+
+   1. Import the `play-services-resolver-1.2.75.0.unitypackage` into your plugin
```

Comment:
Need to ditch the version number here as it's going to get out of date.

Reply:
Whoops. Fixed now.